### PR TITLE
BUG: Preserve output index in `cum_returns`.

### PR DESCRIPTION
**Pull request by ssanderson**
This also now makes all our rolling operations return an output indexed by a suffix of the input index.

### CODE REVIEW

File: empyrical/stats.py [hunk 1]
```diff
@@ -37,7 +37,8 @@ def unary_vectorized_roll(arr, window, out=None, **kwargs):
         window : int
             Size of the rolling window in terms of the periodicity of the data.
         out : array-like, optional
-            The array to store the store the output.
```
Comment (llllllllll) [inline empyrical/stats.py]: This typo...

File: empyrical/tests/test_stats.py [hunk 2]
```diff
@@ -1455,6 +1492,9 @@ class Test2DStatsArrays(Test2DStats):
     def empyrical(self):
         return PassArraysEmpyricalProxy(self, np.ndarray)
 
+    def assert_indexes_match(self, result, expected):
```
Comment (llllllllll) [inline empyrical/tests/test_stats.py]: cute

**Comment by llllllllll** [comment]
lgtm

**Comment by twiecki** [comment]
We should probably cut a new bugfix release.

**Pull request merged**